$DragonFly: src/lib/libmagic/patches/Attic/readelf.h.patch,v 1.1 2005/06/01 15:09:11 joerg Exp $

--- readelf.h	18 Dec 2004 17:18:06 -0000	1.1
+++ readelf.h	31 May 2005 23:00:18 -0000
@@ -223,6 +223,7 @@
 #define NT_NETBSD_EMULATION	2
 #define NT_FREEBSD_VERSION	1
 #define NT_OPENBSD_VERSION	1
+#define NT_DRAGONFLY_VERSION	1
 /* GNU executables (name = "GNU") */
 #define NT_GNU_VERSION		1
 
